egs/iwslt22_ta/ST/pruned_transducer_stateless5/decode.py

```
@@ -648,26 +652,26 @@ def decode_dataset(
         )
-        #breakpoint()
+        # breakpoint()
         for name, hyps in hyps_dict.items():
             this_batch = []
             assert len(hyps) == len(texts)
-            
-            for cut_id, hyp_words, ref_text, ref_text_tgt in zip(cut_ids, hyps, texts, texts_tgt):
+
+            for cut_id, hyp_words, ref_text, ref_text_tgt in zip(
+                cut_ids, hyps, texts, texts_tgt
+            ):
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)